lib/src/base-styles/_nx-page-header.scss

```diff
@@ -39,61 +40,41 @@ $main-header-product-version-row-height: 1fr;
 .nx-product {
   align-self: center;
   box-sizing: border-box;
-  color: $nx-off-white;
-  display: -ms-grid;
-  display: grid;
-  grid-column-gap: $nx-spacing-xxs;
-  -ms-grid-rows: $main-header-product-title-row-height $main-header-product-version-row-height;
-  grid-template-rows: $main-header-product-title-row-height $main-header-product-version-row-height;
-  -ms-grid-columns: $main-header-product-height $nx-spacing-xxs auto;
-  grid-template-columns: $main-header-product-height auto;
-  grid-template-areas:
-    "logo title"
-    "logo version";
+  color: $main-header-color;
+  display: flex;
   height: $main-header-product-height;
   text-decoration: none;
   min-width: $nx-sidebar-width;
 }
 
-.nx-product__logo {
-  -ms-grid-row: 1;
-  -ms-grid-row-span: 2;
-  -ms-grid-column: 1;
-  -ms-grid-row-align: center;
+.nx-product__branding {
   align-self: center;
-  box-sizing: border-box;
-  grid-area: logo;
-  height: 100%;
-  width: 100%;
 }
 
-.nx-product__name {
-  -ms-grid-row: 1;
-  -ms-grid-column: 3;
-  @include semi-bold();
+.nx-product__wordmark {
+  height: 21px;
```

Comment:
Do we have a copy of the image that either fits this height exactly, or is scalable?  The current image is natively 42px tall, which at least is exactly double so the scaling shouldn't be ugly, but I'd think we'd want one that's pre-scaled

Reply:
> Do we have a copy of the image that either fits this height exactly, or is scalable? The current image is natively 42px tall, which at least is exactly double so the scaling shouldn't be ugly, but I'd think we'd want one that's pre-scaled

I got the image directly from @dturner32 I expect the larger size has something to do with how Sketch exports files.

Reply:
> I got the image directly from @dturner32 I expect the larger size has something to do with how Sketch exports files.

I'm not sure what you're trying to say here

Reply:
> > I got the image directly from @dturner32 I expect the larger size has something to do with how Sketch exports files.
> 
> I'm not sure what you're trying to say here

I shrank it.